.storybook/main.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+  webpackFinal(config, options) {
+    config = options.nuxtStorybookConfig.webpackFinal(config, options)
+    return config;
+  },
+
+  addons: [
+    {
+      name: '@storybook/addon-essentials',
+      options: {
+        /**
+         * TODO: Resolve path issues with aliases in docs add-on.
+         * Uncommenting the next line turns the log errors off by deactivating
+         * the docs add-on.
+         */
+        // docs: false,
+      }
+    },
+  ],
+
+  stories: ['../components/**/*.stories.ts'],
```

Comment:
I would suggest the latter, since I can also imaging adding testing files for each component.

Then we would have a structure similar to this:

```
components/
├─ Card/
│  ├─ Card.spec.ts
│  ├─ Card.spec.ts.snap
│  ├─ Card.stories.ts
│  ├─ Card.vue
├─ Link/
│  ├─ Link.spec.ts
│  ├─ Link.spec.ts.snap
│  ├─ Link.stories.ts
│  ├─ Link.vue

```

I think we could benefit from opening a discussion issue on this topic. What do you think? Or maybe @y4izus knows if there is already a plan regarding this?

Reply:
i really like the structure shown above

Reply:
There is no plan regarding this and I like the structure you are proposing @eddybrando. Some thoughts:

- It looks like the way we are doing [global registration of the components](https://github.com/Qiskit/qiskit.org/blob/c032bf258d18ba0020eea6e173691f48a32e8d6e/nuxt.config.ts#L112), implies an auto-import of everything under `~/components/`. We should specify directory paths or file extensions or the combination of both to selectively [include](https://nuxtjs.org/blog/improve-your-developer-experience-with-nuxt-components#inclusion-paths) or [exclude](https://nuxtjs.org/blog/improve-your-developer-experience-with-nuxt-components#exclusion-paths) files from being auto-imported

- I used to use `index.vue` instead of `Card.vue`/`Link.vue`, IMO is a good pattern to follow, what do you all think?

- Please @eddybrando , create the discussion issue you mention :)

Reply:
I'll create an issue to follow up on this here: #1149